bav-ipv-stub/README.md

```diff
@@ -0,0 +1,45 @@
+# IPV Stub
+If there are requirements for data contract changes between IPV Core and CRIs, then please do not use the default stack to build and deploy and instead use a different stack with
+similar naming convention - i.e 
+``` bash
+sam build && sam deploy --stack-name bav-ipv-stub-<YOUR_IDENTIFIER> --resolve-s3 
+```
+
+## Setup notes
+If modifications are made to the test client and a new stack is being used, remember to update the API configuration to use your custom stack.
+
+
+To fetch an ssm parameter created from this stack, use the `get-ssm-test-params.sh` by passing in the key for the ssm parameter.
+
+## Usage
+
+When you deploy this stack you will have access to 3 endpoints.
+
+To start a session use the start endpoint:
+```http 
+POST https://base.uri/stage/start
+Content-Type: application/json
+
+{
+    "target":"<OPTIONAL_TARGET_URI>",
```

Comment:
Just using this for testing the /session endpoint and I've noticed that when I include a target I get a 500 - not sure if that's something to do with the DNS stuff?

<img width="1792" alt="image" src="https://github.com/govuk-one-login/ipv-cri-bav-api/assets/40401118/5f2fc7e9-29ed-4646-a908-b06e2f89896e">

Reply:
Works with the target, might be issue with the DNS urls;
<img width="894" alt="Screenshot 2023-10-17 at 15 35 58" src="https://github.com/govuk-one-login/ipv-cri-bav-api/assets/115095929/846bfb14-8c45-420d-b963-aa77dbc87a62">